src/installer/managed/Microsoft.NET.HostModel/Bundle/BundleOptions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.NET.HostModel.Bundle
+{
+    /// <summary>
+    /// BundleOptions: Optional settings for configuring the type of files
+    ///                included in the single file bundle.
+    /// </summary>
+    [Flags]
+    public enum BundleOptions
+    {
+        None = 0,
+        BundleNativeBinaries = 1,
+        BundleOtherFiles = 2,
+        BundleSymbolFiles = 4,
+        BundleAllContent = BundleNativeBinaries | BundleOtherFiles,
+        ShowDiagnosticTrace = 8
```

Comment:
I would store the `ShowDiagnosticTrace` separately - it is not related to the other values.
I don't mind having more parameters to methods, to provide clarity.